conformance/02_push_test.go

```diff
@@ -204,6 +204,17 @@ var test02Push = func() {
 		})
 
 		g.Context("Cross-Repository Blob Mount", func() {
+			g.Specify("Cross-mounting of a blob without the from argument should yield session id", func() {
+				SkipIfDisabled(push)
+				req := client.NewRequest(reggie.POST, "/v2/<name>/blobs/uploads/",
+					reggie.WithName(crossmountNamespace)).
+					SetQueryParam("mount", dummyDigest)
+				resp, err := client.Do(req)
+				Expect(err).To(BeNil())
+				Expect(resp.StatusCode()).To(Equal(http.StatusAccepted))
+				Expect(resp.GetRelativeLocation()).To(ContainSubstring(crossmountNamespace))
```

Comment:
I think this is not strictly necessary (though I see it in the next case as well :/). The location header may point to an external service which might not include the namespace in the URL. I think we mostly just want to ensure Location is non-empty.